parametric_plasma_source/source_sampling.cpp

```diff
@@ -24,6 +24,7 @@ const double shafranov_shift = 0.0; //metres
 const std::string name = "parametric_plasma_source";
 const int number_of_bins  = 100;
 const int plasma_type = 1; // 1 is default; //0 = L mode anything else H/A mode
+const std::string basis = "xyz"; // xyz for 3D, ry or rz for 2D
```

Comment:
can we tie this to an enum?

Reply:
Makes sense. Linked to an enum now. The enum will need a bit of a tweak when serialising but not major.